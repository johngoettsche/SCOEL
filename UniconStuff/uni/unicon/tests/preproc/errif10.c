/* test error message: no identifier on #ifndef */

#ifndef
one
#endif
