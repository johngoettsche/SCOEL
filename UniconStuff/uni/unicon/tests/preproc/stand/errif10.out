pp: File errif10.c; Line 3: identifier must follow #ifndef
